libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -232,6 +261,20 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     this.#_radioType = value ?? 'info';
   }
 
+  /**
+   * The text to display as the radio button's label. Use this instead of the `sky-radio-label` when the label is text-only.
```

Comment:
remove the second sentence so we don't point consumers to deprecated `sky-radio-label`

Reply:
This is currently how consumers add help text to their form controls, which is why we spell out that `labelText` is for text-only labels. I think this needs to stay until the built-in help widget is added.